tindergames/src/test/java/org/teamcifo/domain/BoardGameTest.java

```diff
@@ -15,11 +15,17 @@ public class BoardGameTest {
     public  void createBoardGameObjectTest(){
         Faker faker = new Faker();
         String gameTitle = faker.backToTheFuture().character();
-        BoardGame game = new BoardGame(gameTitle);
-        assertEquals(gameTitle, game.getGameTitle());
+        BoardGame fakeGame = new BoardGame(gameTitle);
+        assertEquals(gameTitle, fakeGame.getGameTitle());
     }
-
-    public  void createMultipleBoardgames(int number){
+    @Test
+    public void createMultipleBoardGamesTest(){
+        List<BoardGame> boardGameList = createMultipleBoardgames(100);
+        for(BoardGame game : boardGameList){
+            System.out.println(game.toString());
+        }
+    }
```

Comment:
2 things to comment regarding this method:
- Although being a test, it isn't asserting anything, thus not a test at all
- A similar method called `populateBoardGames` is already defined in the `FakeDataGenerator` class inside the test utils package

Please use the above mentioned method instead, and add some kind of assertion to the test. Check the test `printGamesCollection` in the `GamesCollectionManagerTest` for some inspiration on how to check `System.out` prints